src/app/genes/gene-similar/gene-similar.component.html

```diff
@@ -75,9 +75,9 @@ <h1>Similar Genes</h1>
         <ng-template pTemplate="body" let-rowData let-columns="columns">
             <tr [pSelectableRow]="rowData">
                 <td *ngFor="let col of columns">
-                    {{colFinalValue(rowData, col.field, col.subfield)}}
+                    {{colFinalValue(rowData, col.field)}}
```

Comment:
I changed genes-similar to be like genes-list since the latter was in better shape; it might be easier to look at genes-list first.